library/Core/src/Juvix/Core/Erased/Ann/Pretty.hs

```diff
@@ -21,11 +54,19 @@ instance PrimPretty1 primVal => PP.PrettySyntax (Term primTy primVal) where
     Prim p -> fmap HR.toPPAnn <$> PP.pretty' p
     LamM {arguments, body} -> HR.ppLams (arguments, body)
     p@(PairM _ _) -> HR.ppPairs $ getPairs p
+    p@(CatProductIntroM _ _) -> HR.ppPairs $ getPairs p
+    p@(CatProductElimLeftM _ _) -> panic "pretty-printing productelimleft not yet implemented"
```

Comment:
why? seems easy enough

Reply:
I think that was just running out of time!  :) @jonaprieto has some more pretty-printing development in his local repo -- he might be able to let us know whether he has something ready for this case (or might by whenever release hour is).